hazelcast/src/main/resources/META-INF/services/com.hazelcast.DataSerializerHook

```diff
@@ -30,3 +30,4 @@ com.hazelcast.internal.usercodedeployment.impl.UserCodeDeploymentSerializerHook
 com.hazelcast.aggregation.impl.AggregatorDataSerializerHook
 com.hazelcast.projection.impl.ProjectionDataSerializerHook
 com.hazelcast.config.ConfigDataSerializerHook
+com.hazelcast.journal.EventJournalDataSerializerHook
```

Comment:
Missing newline at EOF (violates some POSIX rules, which is a reason for an according CheckStyle rule, which of course doesn't check this file)